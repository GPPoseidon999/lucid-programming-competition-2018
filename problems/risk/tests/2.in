10
1000 1000
1000 998
1000 996
1000 994
1000 992
1000 1000
992 1000
998 1000
996 1000
994 1000
