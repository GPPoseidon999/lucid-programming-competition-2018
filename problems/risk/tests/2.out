0.9988387
0.9989751
0.9990967
0.9992051
0.9993015
0.9988387
0.9980326
0.9986723
0.9984842
0.9982719
